src/Framework/Config/ConfigReader/PhpConfigReader.php

```diff
@@ -52,4 +57,9 @@ private function canRead(string $absolutePath): bool
 
         return $extension === 'php' && file_exists($absolutePath);
     }
+
+    private function triggerEvent(GacelaEventInterface $event): void
```

Comment:
Can we rename this to `dispatchEvent()`?

Reply:
Absolutely ! 👍 💯